__tests__/app.test.js

```diff
@@ -66,4 +64,42 @@ describe("GET /api/reviews", () => {
             expect(reviews).toBeSortedBy('created_at', {descending: false});
           });
       });
+});
+
+describe('GET /api/reviews/:review_id', () => {
+    test('responds with the correct review object', () => {
+      return request(app)
+        .get('/api/reviews/1')
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.review).toEqual({
+            review_id: 1,
+            title: 'Agricola',
+            designer: 'Uwe Rosenberg',
+            owner: 'mallionaire',
+            review_img_url:
+              'https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700',
+            review_body: 'Farmyard fun!',
+            category: 'euro game',
+            created_at: expect.any(String),
+            votes: 1
```

Comment:
It's better to test these properties (except ID!) with a flexible matcher like expect.any(<datatype>) or a .toMatchObject - because then it's less brittle and will work even if the data changes